app/Http/Controllers/ProjectController.php

```diff
@@ -71,8 +78,12 @@ public function store(Request $request)
      */
     public function show($id)
     {
-        $project = Project::find($id);
-        return response()->json($project);
+      $project = Project::find($id);
+      $member_roles = DB::table('project_member')
```

Comment:
If you define the relationship between `project_member` and `member` in your models, you don't need this query.
https://laravel.com/docs/5.5/eloquent-relationships